xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/environment/RippledContainer.java

```diff
@@ -99,9 +100,9 @@ public RippledContainer start() {
   /**
    * Start contain with given interval for closing ledgers.
    *
-   * @param acceptIntervalMillis
+   * @param acceptIntervalMillis the startup time limit.
```

Comment:
This is actually the interval at which we close ledgers in the rippled container, so I would say something like `The number of milliseconds between ledger version closes`